vunit/sim_if/incisive.py

```diff
@@ -38,22 +38,7 @@ class IncisiveInterface(SimulatorInterface):  # pylint: disable=too-many-instanc
 
     sim_options = [ListOfStringOption("incisive.irun_sim_flags")]
 
-    @staticmethod
-    def add_arguments(parser):
-        """
-        Add command line arguments
-        """
-        group = parser.add_argument_group("Incisive irun", description="Incisive irun-specific flags")
-        group.add_argument(
-            "--cdslib",
-            default=None,
-            help="The cds.lib file to use. If not given, VUnit maintains its own cds.lib file.",
-        )
-        group.add_argument(
-            "--hdlvar",
-            default=None,
-            help="The hdl.var file to use. If not given, VUnit does not use a hdl.var file.",
-        )
+    # NOTE: Incisive shares the command-line arguments with Xcelium
```

Comment:
This comment relates to the code that was removed, this should be in a commit message instead. I personally don't think this comment is needed, but I would clean it up to include `add_argument is excluded because X`

Reply:
IMO we could even remove all traces of Incisive, its last release was in 2015, everything after that was called Xcelium.

Reply:
Would the Incisive from 2015 work with the new Xcelium interface?

Reply:
I honestly don't know. Our codebase is incompatible with Incisive due to reliance on newer System Verilog features.